bigflow/bigquery/dataset_configuration.py

```diff
@@ -14,7 +14,7 @@ def __init__(self,
                  is_master: bool = True,
                  is_default: bool = True):
        all_properties = (properties or {}).copy()
-       all_properties['project_id'] = project_id
+       all_properties['project_idxxx'] = project_id
```

Comment:
Probably project_id

Reply:
Sure, it is intentional break (to see broken build).

Reply:
Ok 👍